core/shared/src/main/scala/com/lynbrookrobotics/potassium/streams/ZippedStream.scala

```diff
@@ -1,8 +1,27 @@
 package com.lynbrookrobotics.potassium.streams
 
-import com.lynbrookrobotics.potassium.clock.Clock
+import squants.time.Time
+
+class ZippedStream[A, B](parentA: Stream[A], parentB: Stream[B], skipTimestampCheck: Boolean) extends Stream[(A, B)] {
+  var parentAUnsubscribe: Cancel = null
```

Comment:
Why null and not option

Reply:
This is a very hot part, we don't want to allocate anything ever for sure.